### ISSUE TITLE
[Bug]:crashes when trying to load into the game

### ISSUE THREAD

**Issue by AlastorMeister** [LukeYui/EldenRingSeamlessCoopRelease#604]
### Discord Username

AlastorMiser#2229

### Describe the issue in detail

Every time i try to load into the game after selecting my character profile it crashes. If i delete my dave files and relaunch i can create a new character and go through the tutorial area but if i quit the game and try to relaunch it will crash when trying to load the game.

### Steps to recreate the bug

Not sure if you will be able to recreate it because the game crashes just from trying to load the saved game.

### Version

1.5.1 (Latest)

### Relevant Crash log

```shell
Faulting application name: eldenring.exe, version: 1.9.0.0, time stamp: 0x6405b0f3
Faulting module name: elden_ring_seamless_coop.dll, version: 0.0.0.0, time stamp: 0x641cc4e0
Exception code: 0xc0000409
Fault offset: 0x000000000013bb99
Faulting process id: 0x492c
Faulting application start time: 0x01d964ca7e3a1ab9
Faulting application path: D:\SteamLibrary\steamapps\common\ELDEN RING\Game\eldenring.exe
Faulting module path: D:\SteamLibrary\steamapps\common\ELDEN RING\Game\SeamlessCoop\elden_ring_seamless_coop.dll
Report Id: 051536f6-9739-482d-bbd8-bcd6a258b3fa
Faulting package full name: 
Faulting package-relative application ID:
```

**Comment by Sniderr79** [comment]
nyone is able to assist please do.

**Comment by brainManywrinkle** [comment]
my friend is having that issue too, i think its tied to the custom password inside the .ini file

**Comment by LukeYui** [comment]
Hi, thanks for the report. Please upload your full crash report so I can fix the issue. (located in: \Game\SeamlessCoop\crashdumps\reports)

**Comment by AlastorMeister** [comment]
> Hi, thanks for the report. Please upload your full crash report so I can fix the issue. (located in: \Game\SeamlessCoop\crashdumps\reports)

I no longer have the crash reports. However I may have fixed it on my end, i ended up deleting and uninstalling everything releated to Elden Ring then reinstalled everything and it appears to be working now. I did this because when i have tried playing the unmodded version of elden ring i was getting the same problem.

**Comment by brainManywrinkle** [comment]
I noticed that the new update to Elden ring and the new mod seem to corrupt each other some how? I recreated this by having a currently installed vesion of elden ring, update to version 1.09, install seamless, make character, boom corrupted files make the game crash to desktop. Also! I noticed the loading screen will show the weird grace looking emblem in the bottom left, normally you get a loading bar and tips. This usually happens on new characters but not save stated ones with progress

**Comment by kasanari** [comment]
I believe I encountered this issue. The game crashed a few minutes after loading into the game. Tried load another character. Still crashed. Made a new character. Crashed after loading from the character creator. 

Based on the comments in this thread, I removed the co-op password string, loaded my old character, and it seemed to resolve the issue, at least temporarily. The game did not immediately crash after I changed it.

I am including the dump files that were collected over the hour I tried some different things:
[reports.zip](https://github.com/LukeYui/EldenRingSeamlessCoopRelease/files/11177185/reports.zip)
